f/connectors/globalforestwatch/README.md

```diff
@@ -0,0 +1,101 @@
+# `gfw_alerts`: Download and Store Alerts from Global Forest Watch
+
+This script can retrieve various types of alerts, such as GLAD and Integrated Forest Alerts, from the [Global Forest Watch Data API](https://data-api.globalforestwatch.org/), for a given bounding box. The alerts are then formatted as GeoJSON and stored in a PostgreSQL database. The script also saves the GeoJSON files to a specified directory for further use.
+
+Currently, we support fetching the following alerts from GFW:
+
+* [Integrated deforestation alerts](https://data.globalforestwatch.org/datasets/gfw::integrated-deforestation-alerts/about)
+* [GLAD alerts](https://glad.umd.edu/dataset/glad-forest-alerts) - either Landsat or Sentinel-2
+* [RADD alerts](https://data.globalforestwatch.org/datasets/gfw::deforestation-alerts-radd/about)
+
+To use this script, **you need a valid API key from the GFW Data API**. However, you must first obtain a JWT from the Resource Watch API — this token is then used to authenticate your request to retrieve the GFW Data API key. See [Getting a GFW API Key](#getting-a-gfw-api-key) below.
```

Comment:
This reads strange to me.  Actually I think it's sufficient to just link to the step-by-step below and eschew the nitty-gritty here.

```suggestion
To use this script, **you need a valid API key from the GFW Data API**: See [Getting a GFW API Key](#getting-a-gfw-api-key) below.
```